app/src/ActionModal/components/ModalVoteSubstrate.vue

```diff
@@ -0,0 +1,428 @@
+          field-id="lockedBalance"
+        >
+          <TmField
+            id="lockedBalance"
+            v-model="lockedBalance"
+            v-focus
+            class="locked-balance"
+            type="text"
+            :placeholder="lockedBalance"
+          />
+          <TmFormMsg
+            v-if="$v.lockedBalance.$error && !$v.lockedBalance.required"
+            name="Balance"
+            type="required"
+          />
+          <TmFormMsg
+            v-if="$v.lockedBalance.$error && !$v.lockedBalance.decimal"
+            name="Balance"
+            type="numeric"
+          />
+          <TmFormMsg
+            v-else-if="$v.lockedBalance.$error && !$v.lockedBalance.max"
+            type="custom"
+            :msg="`You don't have enough ${currentNetwork.stakingDenom}s to proceed.`"
+          />
+          <TmFormMsg
+            v-if="$v.lockedBalance.$error && !$v.lockedBalance.min"
+            name="Balance"
+            type="min"
+          />
+          <TmFormMsg
+            v-if="$v.lockedBalance.$error && !$v.lockedBalance.maxDecimals"
+            name="Balance"
+            type="maxDecimals"
+          />
+        </TmFormGroup>
+      </div>
+      <span
+        >{{ currentNetwork.stakingDenom.concat(`s`) }} locked for
+        {{ lockingPeriod }} days</span
+      >
+      <div class="flex-row locking-options">
+        <div
+          v-for="lockingOption in lockingOptions"
+          :key="lockingOption.display"
+        >
+          <span
+            :class="{
+              activeLocking:
+                lockingOption.display === selectedLockingOption.display,
+            }"
+            class="locking-option"
+            @click="lockingOptionController(lockingOption)"
+            >{{ lockingOption.display }}</span
+          >
+        </div>
+      </div>
+    </div>
+    <div class="totals flex-row">
+      <div class="card">
+        <span class="card-title">Total</span>
+        <span>{{ totalVotingPower }}</span>
+        <span>&nbsp;{{ currentNetwork.stakingDenom }}</span>
+      </div>
+    </div>
+    <div class="buttons">
+      <!-- <TmBtn value="Send your Vote" /> -->
+    </div>
+    <TmFormMsg
+      v-if="$v.vote.$error && !$v.vote.required"
+      name="Vote"
+      type="required"
+    />
+  </ActionModal>
+  <!-- </SessionFrame> -->
+</template>
+<script>
+import { mapGetters } from "vuex"
+import { required, decimal } from "vuelidate/lib/validators"
+import { SMALLEST } from "src/scripts/num"
+import ActionModal from "./ActionModal"
+import TmFormGroup from "src/components/common/TmFormGroup"
+import TmField from "src/components/common/TmField"
+// import SessionFrame from "common/SessionFrame"
+import TmBtn from "src/components/common/TmBtn"
+import TmFormMsg from "src/components/common/TmFormMsg"
+import { messageType } from "../../components/transactions/messageTypes"
+import gql from "graphql-tag"
+
+const isValid = (option) => option === `Yes` || option === `No`
```

Comment:
is this necessary?

Reply:
This is also how it is done in ModalVote. I guess it is not so useful, this validation?

Reply:
yep, I think it is not useful. the only useful check is, that there is any option selected